lib/livebook/application.ex

```diff
@@ -204,6 +205,22 @@ defmodule Livebook.Application do
     Livebook.Secrets.set_startup_secrets(secrets)
   end
 
+  def create_offline_hub() do
+    if hash = System.get_env("LIVEBOOK_OFFLINE_HUB") do
+      value = Base.url_decode64!(hash, padding: false)
+      [name, teams_key, public_key] = String.split(value, ":", trim: true)
```

Comment:
I'm open to a better suggestion about the env var structure. (I'll add to README after this decision)

Reply:
Maybe we could use multiple env vars? Here are the requirements:

1. We currently only support offline mode but we will have online mode in the future, so we want to reuse most configuration.

2. The public key is only needed on offline mode, so we don't want to bundle it with other options

So we could keep only name and key together:

    LIVEBOOK_TEAMS_HUB=name:teams_key

But given that we will most likely have more options in the future, I would go with one env per value:

    LIVEBOOK_TEAMS_NAME=name
    LIVEBOOK_TEAMS_KEY=teams_key

Then we need one option for the public key. Given it is only needed for offline mode, we could have one of:

1. `LIVEBOOK_TEAMS_OFFLINE_KEY=public_key`
2. `LIVEBOOK_TEAMS_OFFLINE_PUBLIC_KEY=public_key`
3. `LIVEBOOK_TEAMS_OFFLINE=1` and `LIVEBOOK_TEAMS_PUBLIC_KEY=public_key`

I prefer option 1.

Reply:
I'll with the option 1, and the separated env vars